skymodel.py

```diff
@@ -0,0 +1,202 @@
+                                  lsf_gauss_fwhm=5.0, lsf_boxcar_fwhm=5.0,
+                                  observatory='paranal'):
+    """
+    Returns the model atmospheric transmittance curve queried from the SkyCalc
+    Sky Model Calculator. The default parameters used here are the default
+    parameters provided by SkyCalc:
+    http://www.eso.org/observing/etc/doc/skycalc/skycalc_defaults.txt
+
+    Parameters
+    ----------
+    airmass: float (range [1.0,3.0])
+        Airmass. Alt and airmass are coupled through the plane parallel
+        approximation airmass=sec(z), z being the zenith distance
+        z=90°−Alt
+    pwv_mode: str
+        options: ['pwv','season'] (default is 'pwv')
+    season: int
+        Time of year if not in pwv mode.
+        options: [0,1,2,3,4,5,6] (default is 0)
+        (0 = all year, 1 = dec/jan, 2 = feb/mar...)
+    time: int
+        Period of night. options: [0,1,2,3] (default is 0)
+        (0 = all year, 1, 2, 3 = third of night)
+    pwv: float
+        Precipitable Water Vapor (default is 3.5).
+        options: [-1.0,0.5,1.0,1.5,2.5,3.5,5.0,7.5,10.0,20.0]
+    msolflux: float
+        Monthly Averaged Solar Flux, s.f.u float > 0 (default is 130.0)
+    incl_moon: str
+        Flag for inclusion of scattered moonlight. options = ['Y', 'N']
+        (default is 'Y')
+        Moon coordinate constraints: |z – zmoon| ≤ ρ ≤ |z + zmoon| where
+        ρ=moon/target separation, z=90°−target altitude and
+        zmoon=90°−moon altitude.
+    moon_sun_sep: float
+        Degrees of separation between Sun and Moon as seen from Earth
+        (i.e. the "moon phase").
+        options: [0.0,360.0] (default is 90.0)
+    moon_target_sep: float
+        Moon-Target Separation ( ρ )
+        Degrees in range [0.0,180.0] (defualt is 45.0)
+            # degrees float range [-90.0,90.0] Moon Altitude over Horizon
+    moon_alt: float
+        Moon Altitude over Horizon. Degrees in range [-90.0,90.0]
+        (default is 45.0)
+    moon_earth_dist: float
+        Moon-Earth Distance (mean=1) in range [0.91,1.08]
+        (default is 1.0)
+    incl_starlight: str
+        Flag for inclusion of scattered starlight.
+        options: ['Y', 'N'] (default is 'Y')
+    incl_zodiacal: str
+        Flag for inclusion of zodiacal light.
+        options: ['Y', 'N'] (default is 'Y')
+    ecl_lon: float
+        Heliocentric ecliptic in degree range [-180.0,180.0].
+        (default is 135.0)
+    ecl_lat: float
+        Ecliptic latitude in degree range [-90.0,90.0].
+        (default is 90.0)
+
+    incl_loweratm: str
+        Flag for inclusion of molecular emission of lower atmosphere.
+        options: ['Y', 'N'] (default is 'Y')
+    incl_upperatm: str
+        Flag for inclusion of molecular emission of upper atmosphere.
+        options: ['Y', 'N'] (default is 'Y')
+    incl_airglow: str
+        Flag for inclusion of airglow continuum (residual continuum)
+        options: ['Y', 'N'] (default is 'Y')
+
```

Comment:
Minor: there are a couple of extraneously blank lines in your docstring that can be removed.